File: SofiForce.Models/Models/EntityModels/SalesOrderTypeModel.cs
using System;

namespace Models
{
    public class SalesOrderTypeModel
    {
        public int SalesOrderTypeId { get; set; }
        public string SalesOrderTypeCode { get; set; }
        public string SalesOrderTypeNameEn { get; set; }
        public string SalesOrderTypeNameAr { get; set; }
        public string Icon { get; set; }
        public string Color { get; set; }
        public bool? IsActive { get; set; }
        public int? DisplayOrder { get; set; }
        public bool? CanDelete { get; set; }
        public bool? CanEdit { get; set; }
        public int? CBy { get; set; }
        public DateTime? CDate { get; set; }
        public int? EBy { get; set; }
        public DateTime? EDate { get; set; }
    }
}

File: SofiForce.Web/ClientApp/src/app/core/Models/EntityModels/SalesOrderTypeModel.ts
import { SalesOrderModel } from "./salesOrderModel";

export interface SalesOrderTypeModel {
    salesOrderTypeId: number;
    salesOrderTypeCode: string;
    salesOrderTypeNameEn: string;
    salesOrderTypeNameAr: string;
    icon: string;
    color: string;
    isActive: boolean | null;
    displayOrder: number | null;
    canDelete: boolean | null;
    canEdit: boolean | null;
    cBy: number | null;
    cDate: Date | null;
    eBy: number | null;
    eDate: Date | null;
}